Makefile

```diff
@@ -80,6 +80,9 @@ PLATFORM := $(shell uname -s)
 PATH := $(TOOLCHAIN)/bin:$(PATH)
 SAFEPATH := $(TOOLCHAIN)/bin:/usr/local/sbin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin:/mingw/bin/:/c/tools/mingw32/bin:/c/tools/mingw64/bin
 
+ifneq (,$(findstring CYGWIN,$(PLATFORM)))
```

Comment:
This is tested on cygwin, but I don't know what is right on ubuntu, other linux or BSD.
I'm pretty shure they don't need mingw in there path. ;-)
Or doyou mean.
if mingw then oldpath else myshortpath?